File: ios/RNMBX/RNMBXAtmosphereViewManager.m
#import <React/RCTBridgeModule.h>
#import <React/RCTViewManager.h>

@interface RCT_EXTERN_REMAP_MODULE(RNMBXAtmosphere, RNMBXAtmosphereViewManager, RCTViewManager)

RCT_EXPORT_VIEW_PROPERTY(reactStyle, NSDictionary);

@end

File: ios/RNMBX/RNMBXAtmosphereViewManager.swift
@objc(RNMBXAtmosphereViewManager)
class RNMBXAtmosphereViewManager: RCTViewManager {
    @objc
    override static func requiresMainQueueSetup() -> Bool {
        return false
    }
  
    override func view() -> UIView! {
      let atmosphere = RNMBXAtmosphere()
      //atmosphere.bridge = self.bridge
      return atmosphere
    }
}
